### PR TITLE
Clean up the Deep Learning landing page

### PR DESCRIPTION
**Pull request by BenWilson2**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/BenWilson2/mlflow/pull/18820?quickstart=1)

#### Install mlflow from this PR

```
# mlflow
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/18820/merge
# mlflow-skinny
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/18820/merge#subdirectory=libs/skinny
```

For Databricks, use the following command:

```
%sh curl -LsSf https://raw.githubusercontent.com/mlflow/mlflow/HEAD/dev/install-skinny.sh | sh -s pull/18820/merge
```

</p>
</details>

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?

Refine the landing page for DL. Fix some invalid logos.

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [X] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [X] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [X] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/evaluation`: MLflow model evaluation features, evaluation metrics, and evaluation workflows
- [ ] `area/gateway`: MLflow AI Gateway client APIs, server, and third-party integrations
- [ ] `area/prompts`: MLflow prompt engineering features, prompt templates, and prompt management
- [ ] `area/tracing`: MLflow Tracing features, tracing APIs, and LLM tracing functionality
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/build`: Build and test infrastructure for MLflow
- [X] `area/docs`: MLflow documentation pages

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [X] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

#### Should this PR be included in the next patch release?

`Yes` should be selected for bug fixes, documentation updates, and other small changes. `No` should be selected for new features and larger changes. If you're unsure about the release classification of this PR, leave this unchecked to let the maintainers decide.

<details>
<summary>What is a minor/patch release?</summary>

- Minor release: a release that increments the second part of the version number (e.g., 1.2.0 -> 1.3.0).
  Bug fixes, doc updates and new features usually go into minor releases.
- Patch release: a release that increments the third part of the version number (e.g., 1.2.0 -> 1.2.1).
  Bug fixes and doc updates usually go into patch releases.

</details>

- [X] Yes (this PR will be cherry-picked and included in the next patch release)
- [ ] No (this PR will be included in the next minor release)

### CODE REVIEW

File: docs/static/images/logos/TensorFlow-logo.svg
Comment (harupy) [inline docs/static/images/logos/TensorFlow-logo.svg:1]: what changed?

Comment (harupy) [inline docs/static/images/logos/TensorFlow-logo.svg:1]: These logos look like this in dark mode:

image

Comment (harupy) [reply docs/static/images/logos/TensorFlow-logo.svg:1]: the spacy and tensorflow logos needs some padding like the huggingface logo

Comment (BenWilson2) [reply docs/static/images/logos/TensorFlow-logo.svg:1]: I'll see if I can make the background transparent on all of them. They've been like this for years.

File: docs/docs/classic-ml/deep-learning/index.mdx
Comment (harupy) [inline docs/docs/classic-ml/deep-learning/index.mdx:1]: On mobile, it looks like this. Can we adjust the tile component?

image

File: docs/static/images/logos/TensorFlow-logo.svg
Comment (BenWilson2) [reply docs/static/images/logos/TensorFlow-logo.svg:1]: The logo was missing sizing criteria in the .svg metadata meaning that it couldn't be included in our components. The modification was to add the missing critical metadata so that the .svg didn't get css overrides by our components that resized it to 0px x 0px due to overflow of the flex container.

File: docs/docs/classic-ml/deep-learning/index.mdx [hunk 1]
```diff
@@ -3,420 +3,124 @@ sidebar_position: 0
...
+    image="/images/logos/keras-logo.svg"
+    title="Keras Guide"
+    description="Use Keras 3.0 multi-backend capabilities with unified MLflow tracking."
+    href="/ml/deep-learning/keras"
+    linkText="View guide →"
+    containerHeight={64}
+  />
+  <TileCard
+    image="/images/logos/spacy-logo.svg"
+    title="spaCy Guide"
```
Comment (harupy) [inline docs/docs/classic-ml/deep-learning/index.mdx:45]: Not sure if we still want to highlight spacy. It's dead:

image

File: docs/docs/classic-ml/deep-learning/index.mdx
Comment (BenWilson2) [reply docs/docs/classic-ml/deep-learning/index.mdx:1]: Looks like we had an inline CSS prop on these... adjusted for proper layout in latest push!

File: docs/static/images/logos/TensorFlow-logo.svg
Comment (BenWilson2) [reply docs/static/images/logos/TensorFlow-logo.svg:1]: All 4 fixed!

File: docs/src/components/TileCard/styles.module.css [hunk 2]
```diff
@@ -77,3 +76,39 @@
...
+/* HuggingFace/Transformers logo - now using proper dark mode SVG */
+[data-theme='dark'] .tileCard[href*='transformers'] .tileImage {
+  background: transparent;
+}
+
+/* Keras logo - may have dark elements */
+[data-theme='dark'] .tileCard[href*='keras'] .tileImage {
+  filter: invert(1) hue-rotate(180deg);
+  background: transparent;
+}
```
Comment (harupy) [inline docs/src/components/TileCard/styles.module.css]: This doesn't look like keras. The keras logo looks fine on both light and dark theme.

image

**Review by harupy — APPROVED** [review]
LGTM once https://github.com/mlflow/mlflow/pull/18820/files#r2521720601 is addressed!

File: docs/src/components/TileCard/styles.module.css (hunk 2 above)
Comment (BenWilson2) [reply docs/src/components/TileCard/styles.module.css]: Will revert the color inversion for this logo :)

File: docs/docs/classic-ml/deep-learning/index.mdx (hunk 1 above)
Comment (BenWilson2) [reply docs/docs/classic-ml/deep-learning/index.mdx:45]: Yeah I'll handle this in a separate PR (delisting it as a 'named' flavor and removing it from the tile card entry)
